iroh-rpc-types/src/addr.rs

```diff
@@ -114,6 +127,18 @@ mod tests {
         assert_eq!(addr.to_string(), "grpc://198.168.2.1:1234");
     }
 
+    // TODO(b5): only running this on unix b/c windows doesn't have localhost
+    // enabled by default
```

Comment:
you could start by doing the localhost lookup locally and exit the test early if it doesn't resolve.  that way the test can run everywhere and will do something useful if localhost exists.